app_dart/bin/yaml_converter.dart

```diff
@@ -30,6 +29,15 @@ void writeYaml(SchedulerConfig config) {
   for (Target target in config.targets) {
     configYaml.add('  - name: ${target.name}');
     configYaml.add('    builder: ${target.builder}');
+    if (target.enabledBranches.isNotEmpty) {
```

Comment:
how about adding a single test for this file that verifies the output? That way in code review we can easily preview the diff a change here would make to the generated output.

Reply:
Good point, i'll add an integration test to validate an empty LSC returns the same Cocoon ci.yaml. I added a boiler plate, but it still needs some work